src/lxc/confile.c

```diff
@@ -1790,6 +1792,23 @@ static int set_config_console_path(const char *key, const char *value,
 	return set_config_path_item(&lxc_conf->console.path, value);
 }
 
+static int set_config_console_rotate(const char *key, const char *value,
+				     struct lxc_conf *lxc_conf, void *data)
+{
+	if (lxc_config_value_empty(value)) {
+		lxc_conf->console.log_rotate = 0;
+		return 0;
+	}
+
+	if (lxc_safe_uint(value, &lxc_conf->console.log_rotate) < 0)
+		return -1;
+
+	if (lxc_conf->console.log_rotate > 1)
```

Comment:
Can you add a LOG or ERROR message here?  I probably would have just set log_rotate = 1 if it was > 1, but that's misleading so what you're doing is probably better.  But you should explain to people why they're getting an error.

Reply:
Right. Will update in a bit.